build.gradle

```diff
@@ -29,7 +29,7 @@ group = 'com.exactpro.th2'
 version = release_version
 
 ext {
-    cradleVersion       = '5.0.0-dev-version-5-2567311063-SNAPSHOT'
+    cradleVersion       = '5.0.2-dev-version-5-5176643613-bf4794b-SNAPSHOT'
```

Comment:
Please use cradle dev release

Reply:
There is not dev release for cradle

Reply:
@Nikita-Smirnov-Exactpro If you insist I will add the corresponding workflow to Cradle repository

Reply:
Could make it to clarify what version do we use?

Reply:
We already have the exact commit that corresponds to the version.
But yes, I will add the dev release workflow to Cradle